Gemfile

```diff
@@ -70,3 +71,8 @@ group :test do
   gem "selenium-webdriver"
   gem "webdrivers"
 end
+
+gem 'htmlbeautifier'
+gem 'ordinare'
```

Comment:
On what purpose we are using these gems?

Reply:
Bhaia, HTML Beautifier normaliser/beautifier for HTML that also understands embedded Ruby. I used this gem so that the .erb files can be formatted easily.
Ordinare helps to organise all the gems in the gemfile alphabetically. In the previous review I was suggested to arrange the gem alphabetically for better tracking when we have a lot of gems. That's why I used it.